pkg/lib/config/web3.go

```diff
@@ -22,16 +22,73 @@ type NFTConfig struct {
 	Collections []string `json:"collections,omitempty"`
 }
 
+type BlockchainType string
+
+const (
+	BlockchainTypeEthereum BlockchainType = "ethereum"
+)
+
+var _ = Schema.Add("BlockchainConfig", `
+{
+	"type": "object",
+	"additionalProperties": false,
+	"properties": {
+		"blockchain": { "type": "string", "enum": ["ethereum"] },
+		"network": { "type": "string" }
+	},
+	"required": ["blockchain", "network"],
+	"allOf": [
+		{
+			"if": {
+				"properties": {
+					"blockchain": {
+						"const": "ethereum"
+					}
+				},
+				"required": ["blockchain"]
+			},
+			"then": {
+				"properties": {
+					"network": {
+						"format": "x_web3_ethereum_chain_id"
+					}
+				},
+				"required": ["network"]
+			}
+		}
+	]
+}
+`)
+
+type BlockchainConfig struct {
+	Blockchain BlockchainType `json:"blockchain,omitempty"`
+	Network    string         `json:"network,omitempty"`
+}
```

Comment:
How are we going to use this config?

Reply:
This will be used for two things:
1. The chain SIWE is signing for 
2. The chain network the nft collections in the list are under